addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -51,18 +51,35 @@
 </template>
 
 <template id="s_newsletter_subscribe_popup" name="Newsletter Popup">
-    <div class="o_newsletter_popup o_snippet_invisible" data-list-id="0"/>
+    <div class="s_popup o_newsletter_popup o_snippet_invisible" data-list-id="0" data-name="Newsletter Popup" data-vcss="001" data-invisible="1">
+        <div class="modal fade d-none s_popup_middle o_newsletter_modal"
+            style="background-color: var(--black-50) !important;"
+            data-show-after="5000"
+            data-display="afterDelay"
+            data-consents-duration="7"
+            data-focus="false"
+            data-backdrop="false"
+            tabindex="-1"
+            role="dialog">
```

Comment:
```suggestion
        <div class="modal fade d-none s_popup_middle o_newsletter_modal"
             style="background-color: var(--black-50) !important;"
             data-show-after="5000" data-display="afterDelay" data-consents-duration="7"
             data-focus="false"data-backdrop="false" tabindex="-1" role="dialog">
```
As you want but at least group some parameter by line, no need of one line per paremeter